src/components/button/Button.jsx

```diff
@@ -0,0 +1,14 @@
+import './Button.css'
+import {ArrowRight} from "@phosphor-icons/react";
+
+function Button({label, variant, onClick, hasArrow}) {
+    return (
+        <button className={variant}
+                onClick={onClick}>
+            {label}
+            {hasArrow === true && <ArrowRight size={25}/>}
```

Comment:
Hier checken jullie expliciet (wat prima is) en het zou ook impliciet kunnen, gezien de `hasArrow` waarde altijd een boolean is. Dus `hasArrow && <ArrowRight size={25}/>` werkt ook :)